src/js/common/schemaform/review/StringField.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import _ from 'lodash/fp';
+
+import {
+  getUiOptions,
+  getWidget,
+  optionsList,
+  defaultFieldValue
+} from 'react-jsonschema-form/lib/utils';
+
+/*
+ * This is a minimal string field implementation for the
+ * review page, so we can define pass custom review widgets
+ * in the config
```

Comment:
> so we can define pass custom review widgets

Perhaps you mean
> so we can pass custom review widgets

Or
> so we can define and pass

It's fiddly, I know, but while I'm here I thought I'd ask.